packages/react-aria-components/stories/Popover.stories.tsx

```diff
@@ -48,3 +48,218 @@ export const PopoverExample = () => (
     </Popover>
   </DialogTrigger>
 );
+
+export const PopoverArrowBoundaryOffsetExample = () => {
+  return (
+    <div style={{display: 'flex', flexDirection: 'column'}}>
+      <div style={{display: 'flex'}}>
+        <div style={{padding: 12}}>
+          <DialogTrigger>
+            <Button style={{width: 200, height: 100}}>Top left</Button>
+            <Popover
+              placement="top left"
+              arrowBoundaryOffset={25}
```

Comment:
Just one suggestion, mind adding a storybook control for this value so we can check how it behaves at extreme values?

Reply:
@LFDanLu thanks for the suggestion! I added what you asked for.


https://github.com/adobe/react-spectrum/assets/71210554/4a77434a-51df-413f-960c-14415ad6fb6a